sqrl-examples/banking/docker-compose.yml

```diff
@@ -25,25 +25,29 @@ services:
     ports:
       - "8081:8081"
     command: jobmanager
+    user: root
     environment:
       - |
         FLINK_PROPERTIES=
         jobmanager.rpc.address: flink-jobmanager
+      - TZ:"America/Los_Angeles"
     volumes:
-      - /Users/matthias/git/sqml/sqrl-examples/banking:/Users/matthias/git/sqml/sqrl-examples/banking
```

Comment:
Is there a better way to solve this rather than hardcoding our respective file systems?